lib/src/err/mod.rs

```diff
@@ -77,7 +77,7 @@ pub enum Error {
 		message: String,
 	},
 
-	/// The wrong number of arguments was given for the specified function
+	/// The wrong quantity or magnitude of arguments was given for the specified function
```

Comment:
Note: despite the comment, this error type was already used for issues other than wrong number of arguments.